src/Microsoft.Health.Dicom.Blob/Registration/DicomServerBuilderBlobRegistrationExtensions.cs

```diff
@@ -17,58 +18,83 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class DicomServerBuilderBlobRegistrationExtensions
     {
-        private const string DicomServerBlobConfigurationSectionName = "DicomWeb:DicomStore";
-
         /// <summary>
-        /// Adds the blob data store for the DICOM server.
+        /// Adds the blob data stores for the DICOM server.
         /// </summary>
         /// <param name="serverBuilder">The DICOM server builder instance.</param>
         /// <param name="configuration">The configuration for the server.</param>
         /// <returns>The server builder.</returns>
-        public static IDicomServerBuilder AddBlobStorageDataStore(this IDicomServerBuilder serverBuilder, IConfiguration configuration)
+        public static IDicomServerBuilder AddDataStores(this IDicomServerBuilder serverBuilder, IConfiguration configuration)
```

Comment:
Can we change this to `AddBlobDataStores`? I think the important thing here is that we are accessing blob storage specifically, and I'd like that to be clear when viewing the registration methods.